distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestOidc.java

```diff
@@ -1307,28 +1309,22 @@ private JSONObject createMasterRealmJsonObject() {
   }
 
   /** Configure the OIDC Handler with the given parameters */
-  private void setConfig(Map<String, Object> params) throws IOException {
+  private void setConfig() throws Exception {
     Configuration config =
         getAdminConfig()
             .getConfiguration("org.codice.ddf.security.handler.api.OidcHandlerConfiguration", null);
 
-    config.update(
-        new Hashtable<String, Object>() {
-          {
-            putAll(params);
-          }
-        });
+    config.update(handlerConfig);
 
-    for (Map.Entry<String, Object> entry : params.entrySet()) {
-      expect("Configs to update")
+    for (Enumeration<String> keys = handlerConfig.keys(); keys.hasMoreElements(); ) {
+      String key = keys.nextElement();
+      expect("Wait for the OIDC Handler Configuration to be updated.")
           .within(2, TimeUnit.MINUTES)
-          .until(
-              () ->
-                  config.getProperties() != null
-                      && (config.getProperties().get(entry.getKey()) == entry.getValue()));
+          .until(() -> config.getProperties().get(key).equals(handlerConfig.get(key)));
     }
 
-    waitForSystemReady();
+    // Wait for the OIDC Handler to test the connection, create the OIDC client etc..
+    Thread.sleep(1000);
```

Comment:
Is there anything we can check to see that those things have been done as opposed to just waiting 1 second? On a suuuuuper slow machine, that may not be enough time.

Reply:
We don't really have an endpoint we can hit to verify that. I guess we can verify that the metadata endpoint on the stub server was hit but that still doesn't guarantee that it's done creating everything. I'm gonna double-check